core/src/test/kotlin/com/ritense/valtimo/camunda/repository/CamundaHistoricProcessInstanceSpecificationHelperIntTest.kt

```diff
@@ -130,9 +145,11 @@ class CamundaHistoricProcessInstanceSpecificationHelperIntTest @Autowired constr
     @Transactional
     @WithMockUser(username = TEST_USER, authorities = [ADMIN])
     fun byEndTimeAfter() {
+        Thread.sleep(1000)
```

Comment:
Mysql saves datatype 'datetime' without miliseconds. Unlike postgres. Comparisons involving datetime on MySQL like `time1 < time2` will often fail because the time1 and time2 are equal

Reply:
I get the problem, but I hate the solution. I'll take a closer look to see if I can brew something without having to block the thread.

Reply:
I looked into https://github.com/awaitility/awaitility. but it doesn't work neatly in this case

Reply:
I tried awaitily as well and concluded the same. But I didn't have enough time to properly check an alternative yet.

Reply:
I tried a few things, but it's finicky to make a solution work with both MySQL and PostgreSQL. One of the issues is that we are not testing a real `byEndTimeAfter', but a `byEndTimeEqualOrAfter`.

We could think of changing the mysql column types: https://stackoverflow.com/a/26299379, but I that's stretching the goal of this PR a bit much. I'll add a new story.